src/content/docs/new-relic-solutions/increase-user-engagement.mdx

```diff
@@ -0,0 +1,196 @@
+metaDescription: Capture developer-grade insights with website performance monitoring (WPM) so you can make data-informed improvements to your site. 
+---
+
+import side1wpm from 'images/tutorial_screenshot-crop1_full-wpm.webp'
+
+import side2wpm from 'images/tutorial_screenshot-crop2_full-wpm.webp'
+
+import side1Bwpm from 'images/tutorial_screenshot-crop1b_full-wpm.webp'
+
+import side2Bwpm from 'images/tutorial_screenshot-crop2b_full-wpm.webp'
+
+import pingWPM from 'images/tutorial_screenshot-crop_ping-wpm.webp'
+
+import cwvWPM from 'images/tutorial_screenshot-crop_cwv-wpm.webp'
+
+import timeWPM from 'images/tutorial_screenshot-crop_cwv-time-wpm.gif'
+
+import contentWPM from 'images/tutorial_screenshot-crop_content-wpm.webp'
+
+import stepOne from 'images/tutorial_screenshot-crop_step1-wpm.webp'
+
+import pageSpeedAPI from 'images/tutorial_screenshot-full_pageSpeed-wpm.gif'
+
+Maintaining a website combines the effort of multiple teams. When outages happen, developer and engineering teams have access to resolve the issue directly, but a digital marketer has a different situation. While your team's success is just as dependent on site performance, you might not have the tools to identify or address any issue yourselves.
+
+Our website performance monitoring (WPM) tool gives you insight about your site's availability and performance. With out of the box alerts and routine checks from our monitors, you'll never be in the dark about user experience. 
+
+<SideBySide>
+    <Side>
+<img
+    title="Your WPM page"
+    alt=""
+    src={side1wpm}
+/>
+    </Side>
+    <Side>
+<img
+    title="Your WPM page"
+    alt=""
+    src={side2wpm}
+/>    
+    </Side>
+</SideBySide>
+
+<SideBySide>
+    <Side>
+<img
+    title="Your WPM page"
+    alt=""
+    src={side1Bwpm}
+/>
+    </Side>
+    <Side>
+<img
+    title="Your WPM page"
+    alt=""
+    src={side2Bwpm}
+/>     </Side>
+</SideBySide>
+
+## Objectives [#objectives]
+
+This doc helps you set up and understand your data, so you can empower your team to take control of your customer experience. In this tutorial, you will:
+
+* Set up monitoring for two pages with website performance monitoring 
+* Learn about the data collected from WPM
+
+## Set up WPM [#setup]
+<Steps>
+    <Step>
+    ### Get data without installation
+
+    To access WPM, go to **[one.newrelic.com > Website performance monitoring](https://one.newrelic.com/website-grader)**.
+
+    </Step>
+    <Step>
+        ### Choose your URL 
+
+    WPM requires no installation or set up. Once you've chosen pages to monitor, New Relic deploys a set of monitors from different servers around the globe that check each page for website availability, broken links, performance, and SSL certificate expiration. To get started, we recommend choosing these pages:
+
+    * Your homepage.
+    * Pages with critical assets, like important videos, images, or links.
+    * Pages critical to your customer funnel
```

Comment:
Pages critical to your customer funnel such as thank you page, product page etc.